tests/network/migration/test_migration.py

```diff
@@ -385,6 +385,7 @@ def test_connectivity_after_migration_and_restart(
 
 
 @pytest.mark.polarion("CNV-2061")
+@pytest.mark.gating
```

Comment:
This test will fail if executed on PSI because those clusters do not support IPv6.
Are you sure we can mark this as gating at the moment?

Reply:
I am sure [you are right](https://jenkins-csb-cnvqe-main.dno.corp.redhat.com/job/openshift-virtualization-tests-runner/2201/testReport/tests.network.migration/test_migration/test_migration_with_masquerade__ipv6__/).
I will remove the marker from this test.